Dockerfile

```
@@ -1,9 +1,10 @@
 FROM java:openjdk-8-jre-alpine
 
-RUN apk update && apk add --no-cache unzip
-RUN adduser -D -h /home/ircbot -u 1013 ircbot
-ADD target/ircbot-2.0-SNAPSHOT-bin.zip /usr/local/bin/ircbot.zip
-RUN cd /usr/local/bin; unzip ircbot.zip
+RUN apk update && apk add --no-cache unzip=6.0-25ubuntu1 && \
+    adduser -D -h /home/ircbot -u 1013 ircbot
+COPY target/ircbot-2.0-SNAPSHOT-bin.zip /usr/local/bin/ircbot.zip
+WORKDIR /usr/local/bin 
+RUN unzip ircbot.zip
```

```suggestion
RUN unzip ircbot.zip -d /usr/local/bin
```
Better to keep the default WORKDIR inherited from the parent image (and /usr/local/bin as workdir does not make sense)

---

there were a `cd` in this folder before, and nothing to get out of it after, but your suggestion is definitively better.